source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOHostConfig.h

```diff
@@ -13,4 +13,12 @@
 - (instancetype)init;
 
 + (ACOHostConfigParseResult *)fromJson:(NSString *)payload;
+
 @end    
+
+
+@interface ACOTextConfig:NSObject
```

Comment:
What's this? It doesn't look like it's implemented anywhere.

Reply:
thank you very much

---
In reply to: [171735944](https://github.com/Microsoft/AdaptiveCards/pull/1178#discussion_r171735944) [](ancestors = 171735944)